sklearn/metrics/_plot/__init__.py

```diff
@@ -0,0 +1,40 @@
+def _check_classifer_response_method(estimator, response_method):
```

Comment:
Is it good to include things in __init__.py? Perhaps base.py?
Let's update plot_roc_curve in this PR?